src/store/plugins/fungibleTokens.js

```diff
@@ -60,17 +62,18 @@ export default (store) => {
       },
     },
     actions: {
-      async getAvailableTokens({ rootGetters: { activeNetwork }, commit }) {
-        const response = await fetchJson(
-          `${activeNetwork.middlewareUrl}/aex9/by_name`,
-        ).catch(handleUnknownError);
-
-        if (isEmpty(response) || typeof response !== 'object') return commit('setAvailableTokens', {});
+      async getAvailableTokens({ rootState: { middleware }, commit }) {
+        if (!middleware) return;
+        const result = await getAllPages(() => middleware.fetchByPath('/v2/aex9?by=name&limit=100'), middleware.fetchByPath);
+        if (!result?.length) {
+          commit('setAvailableTokens', {});
+          return;
+        }
 
-        const availableTokens = response.reduce((obj, { contract_id: contract, ...other }) => ({
+        const availableTokens = result.reduce((obj, { contract_id: contract, ...other }) => ({
```

Comment:
This code is not adapted to work with `camelcaseKeysDeep`ed data that is now returning, which results in having wrong data in availableTokens.